.expeditor/config.yml

```diff
@@ -3,11 +3,16 @@
 github:
   # The file where the MAJOR.MINOR.PATCH version is kept. The version in this file
   # is bumped automatically via the `built_in:bump_version` merge_action.
-  version_file: "VERSION"
+  # TODO: Change this to `VERSION` once we are live
+  version_file: "VERSION_FAKE"
```

Comment:
This is so we don't accidentally update the version while iterating on the promotion and acceptance testing.